app/packages/state/src/hooks/useSchemaSettings.ts

```diff
@@ -76,23 +80,31 @@ export const schemaSearchRestuls = atom<string[]>({
         const viewSchema = await getPromise(viewSchemaState);
         const schema = await getPromise(schemaState);
 
-        const greenPaths = [...newPaths].filter(
-          (path) =>
-            path &&
-            (viewSchema?.[path]?.ftype || schema?.[path]?.ftype) &&
-            !skipFiled(path, viewSchema[path].ftype)
-        );
+        const greenPaths = [...newPaths]
+          .filter(
+            (path) =>
+              path &&
+              (viewSchema?.[
+                path.startsWith("frames.") ? path.replace("frames.", "") : path
```

Comment:
This could break image datasets with a `frames` field. There are probably existing issues with a `frames` field on image datasets but just sharing some context. Ok with me until a more formal discussion happens

Reply:
do you mean custom defined `frames` field on image datasets? videos are the only ones with `frames` prefix I thought 🤔

Reply:
I always developed the App to in theory support adding an arbitrary `frames` field, e.g.
```py
dataset.add_sample_field("frames", fo.IntField)
```

But it could very likely not be supported in the SDK / in general. It's just not officially documented. I digress

Reply:
The SDK is careful to allow `frames` fields for image datasets. The `frames` keyword only has special meaning for datasets that contain videos.